client/src/components/employer-component/employer-info-component/index.jsx

```diff
@@ -0,0 +1,172 @@
+  getUserById,
+  managedUsers,
+  setSnackBarMessage,
+}) {
+  const vertButtonRef = useRef(null);
+  const [anchorEl, setAnchorEl] = useState(null);
+  const open = Boolean(anchorEl);
+
+  let owner = getUserById(employer.owner);
+  let creator = getUserById(employer.creator);
+
+  useEffect(() => {
+    owner = getUserById(employer.owner);
+    creator = getUserById(employer.creator);
+  }, [employer]);
+
+  const handleClick = () => {
+    setAnchorEl(vertButtonRef.current);
+  };
+
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+  const [ownerChangeDialog, setOwnerChangeDialog] = React.useState(false);
+  const [error, setError] = React.useState(null);
+
+  const onEditOwnerClick = () => {
+    setOwnerChangeDialog(true);
+  };
+
+  const onOwnerChangeConfirm = () => {
+    setOwnerChangeDialog(false);
+  };
+
+  const onChangeOwnerCancel = () => {
+    setOwnerChangeDialog(false);
+  };
+
+  if (error) return <ErrorScreenComponent message={error} />;
+
+  return (
+    <TopRowContainer
+      style={{ display: "flex", justifyContent: "space-between" }}
+    >
+      <div>
+        <HeaderContainer>
+          <Typography
+            style={{
+              fontFamily: "Arial",
+              fontSize: "34px",
+              fontWeight: 700,
+              lineHeight: "42px",
+              letterSpacing: "0.25px",
+              textAlign: "left",
+            }}
+          >
+            {employer.name}
+          </Typography>
+          <IconButton onClick={handleClick}>
+            <MoreVertIcon
+              sx={{
+                color: "gray",
+                cursor: "pointer",
+              }}
+              ref={vertButtonRef}
+            />
+          </IconButton>
+
+          <Menu
+            anchorEl={anchorEl}
+            open={open}
+            onClose={handleClose}
+            MenuListProps={{
+              "aria-labelledby": "split-button",
+            }}
+            PaperProps={{
+              style: {
+                width: vertButtonRef ? 190 : null,
+              },
+            }}
+          >
+            <MenuItem onClick={handleClose}>
+              <IconButton>
+                <DownloadIcon />
+              </IconButton>{" "}
+              Export as JSON
+            </MenuItem>
+            <MenuItem onClick={handleClose}>
+              <IconButton>
+                <DeleteIcon />
+              </IconButton>
+              Delete
+            </MenuItem>
+          </Menu>
+        </HeaderContainer>
+        <Typography
+          style={{
+            fontFamily: "Arial",
+            textAlign: "left",
+          }}
+        >
+          Employer
+        </Typography>
+      </div>
+
+      <Box
+        sx={{
+          marginLeft: "auto",
+          display: "flex",
+          alignItems: "center",
+          borderRadius: "8px",
+          boxShadow: 2,
+          p: 3,
+          mr: 6.5,
+        }}
+      >
+        <Box sx={{ textAlign: "center", mr: 2 }}>
```

Comment:
We want to make sure this component is right aligned along our margin border for the page similar to the other object pages